src/Orleans.Serialization/Buffers/Writer.cs

```diff
@@ -153,7 +153,7 @@ internal Writer(TBufferWriter output, Span<byte> span, SerializerSession session
 
         /// <inheritdoc/>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public void Dispose()
+        public readonly void Dispose()
```

Comment:
This is odd, since `Output.Dispose()` shouldn't be readonly itself, and it's potentially a struct. I wonder if this is making defensive copies where it shouldn't be.

Reply:
Removed it